src/reconcile/rec_col.c

```diff
@@ -323,6 +339,8 @@ __wt_rec_col_int(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_REF *pageref)
         if (page_del != NULL)
             WT_TIME_AGGREGATE_MERGE(session, &r->cur_ptr->ta, &ft_ta);
         WT_TIME_AGGREGATE_MERGE(session, &r->cur_ptr->ta, &ta);
+        if (__wt_process.page_stats_2022)
```

Comment:
why check the flag here? Based on the flag we do not write to disk already. Is there any specific reason behind it?